index.html

```diff
@@ -2172,6 +2172,18 @@ <h2>Method Schemes</h2>
 formats as possible.
       </p>
 
+      <p>
+The <code>method-specific-id</code> format MAY include colons, which might be
+used by <a>DID methods</a> for various purposes, such as establishing
+hierarchically partitioned namespaces, or identifying specific instances or
+parts of the <a>DID registry</a>. However, as long as the use of colons
+complies syntactically with the <code>method-specific-id</code> ABNF rule,
+their use is entirely method-specific. Implementers are advised to avoid
```

Comment:
> `However, as long as the use of colons complies syntactically with the <code>method-specific-id</code> ABNF rule, their use is entirely method-specific.`

The "as long as" part makes it sound like that's optional.  Can colon use ever not comply with the ABNF rules?  Perhaps this should be more direct (and I'm not sure "method-specific" needs the hyphen):

`The use of colons MUST comply syntactically with the <code>method-specific-id</code> ABNF rule and their use is entirely method specific.`

Reply:
I agree with your suggestion, applied it in d7fc589bd6c22b4d6a58e1bd127cc3c17a69183d

Reply:
Actually I kept the hyphen in "method-specific", since that's how it's written everywhere else in the spec. If we want to change that we should create a separate issue.